api4/cloud.go

```diff
@@ -141,6 +143,33 @@ func getCloudProducts(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Write(json)
 }
 
+func getCloudLimits(c *Context, w http.ResponseWriter, r *http.Request) {
+	if c.App.Channels().License() == nil || !*c.App.Channels().License().Features.Cloud {
+		c.Err = model.NewAppError("Api4.getCloudLimits", "api.cloud.license_error", nil, "", http.StatusNotImplemented)
+		return
+	}
+
+	if !c.App.Config().FeatureFlags.CloudFree {
```

Comment:
Does this feature flag indicate the user is on a cloud-free instance? If so, do we have the ability to change feature flags at runtime?

In Boards should we check for `License().Features.Cloud` and `FeatureFlags.CloudFree` before calling the `/limits` endpoint to avoid erros?

Reply:
This flag only indicates whether the feature is enabled in general. The checks above are like what you mention checking in boards, and make it so that unlicensed or non-cloud licensed instances are still given the "Not Implemented" error response.

Reply:
nit: For ease of implementation in the clients, if the feature flag is not enabled, should we just return empty limits. This way the client doesn't have to check for the feature flag and just act like it has no limits case instead of having to check or handle the error 0/5

Reply:
That sounds good to me for cloud instances. For on-prem instances, I think I'd rather return the not implemented error to encourage clients to not make the request in the first place.

Reply:
Agreed, clients should still check for cloud license which is a permanent check. However, the feature flag will be removed so one less case to remove. Again, 0/5 and possibly a preference for the clients